### ISSUE TITLE
[TC] [AC] [Question] [Group] Achieving blizz like group xp reduction

### ISSUE THREAD

**Issue by Mershl** [trickerer/Trinity-Bots#682]
### DO NOT REMOVE OR SKIP THE ISSUE TEMPLATE

- [X] I understand that my issue may get **closed without notice** if I *intentionally* remove or skip any mandatory\* field
- [X] This is NOT a bug/crash report
- [X] I've searched the bugtracker for similar questions/requests **including closed ones**

### Description

Hi @trickerer,

first of all A HUGE THANK YOU for this project. We're having lots of fun with it so far.

Most of the time we're playing dungeons with 2 players and 3 bots - which brings up a problem: we need to use XpReduction to avoid having the player characters level way too fast in dungeons as XP is calculated as if they would clear the dungeon with 2 characters, but as 1 player will have 2 bots and 1 player will have 1 bot we didn't find a way to have the players level at an equal pace. An option would be to not use XpReduction at all and play with XP.rate serverwide (massivly reducing it when we assume to do a dungeon with 2 players + 3 bots).

Long story short: Is it technically possible to achieve the blizzlike group XP reduction - so that killing a mob with 2players+3bots in a group would give the same xp as killing it with 5players in a group?

Thank you and continue being awesome!

### TrinityCore or AzerothCore

AzerothCore

### Core rev. hash/commit

latest

**Comment by trickerer** [comment]
That's possible, yes

<details>
<summary>Note to Self</summary>
Bot XP reduction should count bots in group, not individually
</details>
